### ISSUE TITLE
Latency numbers every programmer should know update

### ISSUE THREAD

**Issue by abrarisme** [donnemartin/system-design-primer#236]
Isn't the section for **Latency numbers every programmer should know** outdated? I went to the following page to get interactive numbers for 2018: https://people.eecs.berkeley.edu/~rcs/research/interactive_latency.html 

Example:
- reading 1 million bytes from SSD we get 78 microseconds instead of 1ms
- mutex lock/unlock is 17ns instead of 100ns

**Comment by IAmPramod** [comment]
@abrarisme This article is great. 
@donnemartin we should change the latency numbers

**Comment by devdil** [comment]
These numbers are very relative

**Comment by pbanavara** [comment]
If you move the slider to 2019 the figures change a bit. It's 62 microseconds for 1M SSD read. So we need to change the Readme. I am more than happy to do this and submit a PR. Is that how this works ?

**Comment by devdil** [comment]
Yes, Please include block size also. As far my knowledge, thats the typical
latency in SSD (random read) for a 4KB block.

Thanks
Diljit

**Comment by devdil** [comment]
Yes, Please include block size also. As far my knowledge, thats the typical
latency in SSD (random read) for a 4KB block.

Thanks
